target/ppc/int_helper.c

```diff
@@ -1710,12 +1710,11 @@ VEXTRACT(d, u64)
 void helper_##NAME(CPUPPCState *env, ppc_avr_t *t, ppc_avr_t *b,    \
                    target_ulong rc)                                 \
 {                                                                   \
-    const int array_size = ARRAY_SIZE(b->u##SIZE);                  \
     bool null_found = false;                                        \
     int i, idx;                                                     \
                                                                     \
-    for (i = 0; i < array_size; i++) {                              \
-        idx = LEFT ? i : array_size - i - 1;                        \
+    for (i = 0; i < SIZE; i++) {                                    \
```

Comment:
Nit: I suggest renaming SIZE -> NUM_ELEMS, or something similar. Calling it SIZE gives me the (wrong) idea that it should be 8 for B and 16 for H, and not the other way around.

Reply:
Agreed. Fixed in ab0157c9ab4e7015d90e818855827c1a8ca0e648.